.github/workflows/build-builder.yaml

```diff
@@ -0,0 +1,35 @@
+name: build-builder
```

Comment:
missing copyright headers

Reply:
:+1:  btw should I update the dates to 2024?

Reply:
done